bears/js/ESLintBear.py

```diff
@@ -48,7 +48,7 @@ def generate_config(filename, file):
         return '{"extends": "eslint:recommended"}'
 
     def process_output(self, output, filename, file):
-        if not file:
+        if not file or not output:
             return
```

Comment:
i.e. the message will probably have a similar structure, make a regex and match the message?